extensions/adobe/experience/content-telemetry/webpage.schema.json

```diff
@@ -0,0 +1,111 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/content-telemetry/webpage",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Content Telemetry Web Page",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "Web page details and containing web elements",
+  "definitions": {
+    "webpage": {
+      "properties": {
+        "repo:id": {
+          "title": "Page Id",
+          "type": "string",
+          "meta:immutable": true,
+          "meta:usereditable": false,
+          "description": "The unique ID of the web page."
+        },
+        "repo:name": {
+          "meta:usereditable": false,
+          "title": "Page name",
+          "type": "string"
+        },
+        "dc:title": {
+          "title": "Title",
+          "type": "string",
+          "description": "The title of the web page"
+        },
+        "repo:lastModifiedDate": {
+          "type": "string",
+          "format": "date-time",
+          "meta:usereditable": false,
+          "examples": ["2004-10-23T12:00:00-06:00"],
+          "description": "The server date and time when the resource was last modified in the repository, such as when a new version of an asset is uploaded or a directory's child resource is added or removed. The date time property should conform to ISO 8601 standard. An example form is \"2004-10-23T12:00:00-06:00\"."
+        },
+        "xdm:pageType": {
+          "title": "Type",
+          "type": "string",
+          "description": "The type of the component."
+        },
+        "xdm:templatePath": {
+          "title": "Page Template path",
+          "type": "string",
```

Comment:
Should this also have a `format:uri`?

Reply:
the value is something like:
```
/content/core-components-examples/library/jcr:content/root/responsivegrid/text
```
which is not validated by `format:uri`